PEReaderAndWriter/PEReader/BinaryObjectModel.cs

```diff
@@ -492,7 +492,7 @@ ModuleIdentity moduleIdentity
     }
 
     bool IModule.Requires32bits {
-      get { return (this.Cor20Flags & (COR20Flags.Bit32Required|COR20Flags.Prefers32bits)) == COR20Flags.Bit32Required; }
```

Comment:
I believe this was correct originally. According to https://github.com/dotnet/coreclr/blob/5304962d18bccdfe84f08c7ffb1230c3c4d1763a/src/inc/corhdr.h If both bits are set, then the image is platform neutral, but prefers 32-bit when possible. This property implies that the image must be 32-bits, not just prefers it.

Reply:
Ah, I see. Thanks! I can revert this change.